requirements_frozen.txt

```diff
@@ -1,3 +1,4 @@
+wheel==0.36.2
```

Comment:
It would be nice to sort these two files and keep them alphabetized, but it isn't currently, so I guess it is not required for this change.